### DISCUSSION TITLE
Does this task support incremental deployment?

### DISCUSSION THREAD

**Discussion by meghakhan95** [Azure-Player/azure.datafactory.tools#237] [Q&A] [answered]
Hello Team,

A customer of mine is using this task to deploy ADF resources via azure pipelines. But there is no way to know if it supports incremental deployment. Could you please confirm if it does incremental deployment or complete deployment each time?

**Comment by NowinskiK** [comment, marked as answer]
Hi @meghakhan95,
Thanks for asking. 
The tool does complete deployment every time, so no incremental deployment is supported,
however the are some plans for that: #195
